mitmproxy/addons/onboarding.py

```diff
@@ -6,7 +6,7 @@
 APP_PORT = 80
 
 
-class Onboarding(asgiapp.WSGIApp):
+class Onboarding(asgiapp.ASGIApp):
```

Comment:
Our onboarding app is wsgi, so this breaks http://mitm.it/. Just changing it back to WSGI works for me.

Reply:
OK. I don't know why ASGIApp worked for me, but I'll change it back.